src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AcrylicBrush.Rendering.Blur.Android.cs

```diff
@@ -0,0 +1,267 @@
+﻿using System;
+using System.Drawing;
+using Android.Content;
+using Android.OS;
+using Android.Views;
+using Android.Widget;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.UI;
+using Uno.UI.Xaml.Media;
+using View = Android.Views.View;
+
+namespace Windows.UI.Xaml.Media
+{
+	public partial class AcrylicBrush
+	{
+		private const float AndroidBlurRadius = 20;
+
+		private const float StyledBlurRadius = 64;
+
+		private static readonly Color DarkBlurOverlayColor = Color.FromArgb(128, 0, 0, 0);
+
+		//private static readonly Color LightBlurOverlayColor = Color.FromHex("#40FFFFFF");
+
+		//private static readonly Color ExtraLightBlurOverlayColor = Color.FromHex("#B0FFFFFF");
+
+		private static int blurAutoUpdateDelayMilliseconds = 100;
+		private static int blurProcessingDelayMilliseconds = 10;
+
+		private RealtimeBlurView _realtimeBlurView;
+
+		//private View _blurRootView;
+
+		/// <summary>
+		/// When a page visibility changes we activate or deactivate blur updates.
+		/// Setting a bigger delay could improve performance and rendering.
+		/// </summary>
+		public static int BlurAutoUpdateDelayMilliseconds
+		{
+			get => blurAutoUpdateDelayMilliseconds;
+			set
+			{
+				if (value < 0)
+				{
+					throw new ArgumentException(
+						"The blur processing delay cannot be negative",
+						nameof(BlurAutoUpdateDelayMilliseconds));
+				}
+
+				blurAutoUpdateDelayMilliseconds = value;
+			}
+		}
+
+		/// <summary>
+		/// Sometimes the computation of the background can take some times (svg images for example).
+		/// Setting a bigger delay to be sure that the background is rendered first can fix some glitches.
+		/// </summary>
+		public static int BlurProcessingDelayMilliseconds
+		{
+			get => blurProcessingDelayMilliseconds;
+			set
+			{
+				if (value < 0)
+				{
+					throw new ArgumentException(
+						"The blur processing delay cannot be negative",
+						nameof(BlurProcessingDelayMilliseconds));
+				}
+
+				blurProcessingDelayMilliseconds = value;
+			}
+		}
+
+		/// <summary>
+		/// If set to <see langword="true"/>, the rendering result could be better (clearer blur not mixing front elements).
+		/// However due to a bug in the Xamarin framework https://github.com/xamarin/xamarin-android/issues/4548, debugging is impossible with this mode (causes SIGSEGV).
+		/// A suggestion would be to set it to false for debug, and to true for releases.
+		/// </summary>
+		public static bool ThrowStopExceptionOnDraw { get; set; } = false;
+
+		private bool IsAndroidBlurPropertySet => AndroidBlurRadius > 0;
+
+		private double CurrentBlurRadius =>
+			IsAndroidBlurPropertySet ? AndroidBlurRadius : StyledBlurRadius;
+
+		//TODO
+		//protected void OnAttachedToWindow()
+		//{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)